README.md

```diff
@@ -94,8 +94,13 @@ There are a few restrictions when using this library on stable Rust:
 - You will have to use the `const_*` functions (e.g. `const_mutex(val)`) to
   statically initialize the locking primitives. Using e.g. `Mutex::new(val)`
   does not work on stable Rust yet.
-- The `wasm32-unknown-unknown` target is only supported on nightly and requires
-  `-C target-feature=+atomics` in `RUSTFLAGS`.
+- The `wasm32-unknown-unknown` target is only fully supported on nightly with
+  `-C target-feature=+atomics -Z build-std` in `RUSTFLAGS`.
```

Comment:
`-Zbuild-std` is an argument to be passed directly to cargo or enabled in `.cargo/config.toml`. `-Ctarget-feature=+atomics` is an argument of rustc passed using the `RUSTFLAGS` env var.

Reply:
ah, thanks, I split them up accordingly.